interfaces/builtin/broadcom_asic_control.go

```diff
@@ -49,9 +40,11 @@ const broadcomAsicControlConnectedPlugAppArmor = `
 `
 
 const broadcomAsicControlConnectedPlugUDev = `
-KERNEL=="linux-user-bde", TAG+="###SLOT_SECURITY_TAGS###"
-KERNEL=="linux-kernel-bde", TAG+="###SLOT_SECURITY_TAGS###"
-KERNEL=="linux-bcm-knet", TAG+="###SLOT_SECURITY_TAGS###"
+# This file contains udev rules for accessing broadcom asic kernel module.
+#
+# Do not edit this file, it will be overwritten on updates
+
+SUBSYSTEM=="net", KERNEL=="bdev", TAG+="%[1]s"
```

Comment:
Yes, basically, this interface was added since it's needed by a customer project.
I compared the rule with what was being added into upstream and found the difference here.
I'll do a further investigation on these two rules and confirm the final behavior on real device and will add the upstream link above the rule if it's the one we needed.